shell/platform/darwin/macos/framework/Source/FlutterAccessibilityMac.mm

```diff
@@ -0,0 +1,575 @@
+      screen_bounds.size.height - global_bounds.origin.y - global_bounds.size.height;
+  SkRect result = SkRect::MakeXYWH(global_bounds.origin.x, global_bounds.origin.y,
+                                   global_bounds.size.width, global_bounds.size.height);
+  return result;
+}
+
+// Private method
+std::string FlutterAccessibilityMac::GetLiveRegionText() const {
+  if (GetAXNode()->IsIgnored())
+    return "";
+
+  std::string text = GetData().GetStringAttribute(ax::StringAttribute::kName);
+  if (!text.empty())
+    return text;
+
+  for (int32_t child : GetData().child_ids) {
+    const FlutterAccessibilityMac* accessibility_child =
+        (FlutterAccessibilityMac*)GetBridge()->GetFlutterAccessibilityFromID(child);
+    if (!accessibility_child)
+      continue;
+
+    text += accessibility_child->GetLiveRegionText();
+  }
+  return text;
+}
+
+void FlutterAccessibilityMac::computeTextEdit(std::u16string& inserted_text,
+                                              std::u16string& deleted_text,
+                                              id* edit_text_marker) {
+  if (!GetData().IsTextField())
+    return;
+
+  // Starting from macOS 10.11, if the user has edited some text we need to
+  // dispatch the actual text that changed on the value changed notification.
+  // We run this code on all macOS versions to get the highest test coverage.
+  std::u16string old_value = old_text_editing_value_;
+  std::wstring_convert<std::codecvt_utf8_utf16<char16_t>, char16_t> convert;
+  std::u16string new_value = convert.from_bytes(GetAXNode()->GetValueForControl());
+  old_text_editing_value_ = new_value;
+  if (old_value.empty() && new_value.empty())
+    return;
+
+  size_t i;
+  size_t j;
+  // Sometimes Blink doesn't use the same UTF16 characters to represent
+  // whitespace.
+  for (i = 0; i < old_value.length() && i < new_value.length() &&
+              (old_value[i] == new_value[i] || (iswspace(old_value[i]) && iswspace(new_value[i])));
+       ++i) {
+  }
+  for (j = 0; (i + j) < old_value.length() && (i + j) < new_value.length() &&
+              (old_value[old_value.length() - j - 1] == new_value[new_value.length() - j - 1] ||
+               (iswspace(old_value[old_value.length() - j - 1]) &&
+                iswspace(new_value[new_value.length() - j - 1])));
+       ++j) {
+  }
+  FML_DCHECK(i + j <= old_value.length());
+  FML_DCHECK(i + j <= new_value.length());
+
+  std::u16string current_deleted_text = old_value.substr(i, old_value.length() - i - j);
+  std::u16string current_inserted_text = new_value.substr(i, new_value.length() - i - j);
+
+  // Heuristic for editable combobox. If more than 1 character is inserted or
+  // deleted, and the caret is at the end of the field, assume the entire text
+  // field changed.
+  // TODO(nektar) Remove this once editing intents are implemented,
+  // and the actual inserted and deleted text is passed over from Blink.
+  if (GetData().role == ax::Role::kTextFieldWithComboBox &&
```

Comment:
This TODO has no relevance to Flutter, right?

Do we need our own todo for htis, or should we just remove it?

Reply:
I'd err on the side of leaving comments like this in the code, as it gives us context surrounding the design and/or future considerations.